src/Inviqa/Worldpay/Api/Request/AuthorizeRequestFactory.php

```diff
@@ -201,4 +235,60 @@ public function buildFromRequestParameters(array $parameters): PaymentService
 
         return $paymentService;
     }
+
+    /**
+     * @param array $parameters
+     *
+     * @return PaymentService
+     * @throws InvalidRequestParameterException
+     */
+    public function buildApplePayFromRequestParameters(array $parameters): PaymentService
```

Comment:
This class is now quite large.
Is it worth splitting it down into multiple factories? e.g. have a separate one just for the ApplePay parameters?